src/initialization.js

```diff
@@ -21,20 +22,18 @@ var initialization = {
         processEvent,
         eventQueue,
         isInitialized,
-        common,
-        appVersion,
-        appName,
-        customFlags,
-        clientId
+        common
```

Comment:
i know aI'd either leave the remaining arguments here along with an eslint disable before it for unused arguments, or a comment saying what other arguments are available.  The likeliest one that may be used in the future is `customFlags`.  If someone in the future came back to look at this, it may be hard for them to know where that should come from without a comment or the argument here.

Reply:
I think a comment works best as I don't want someone to forget to turn off the eslint disable setting in the future. Some of this feedback indicates that we should take a look at the example repo and consider adding something like JSDocs to declare what parameters are available for derived kits instead of adding unnecessary variables that could cause some code bloat.